packages/teams-js/src/public/appInitialization.ts

```diff
@@ -1,59 +1,59 @@
 import { app } from './app';
 
 /**
- * @deprecated with TeamsJS v2 upgrades
+ * @deprecated As of 2.0.0-beta.1. Please use {@link app} namespace instead.
```

Comment:
Can you please update these as per Erika's suggestion: e.g.,

`As of 2.0.0-beta.1, please use {@link app} namespace instead.`

Note the "... , please ..." instead of two separate sentences.